src/tools/rados/rados.cc

```diff
@@ -2870,10 +2871,10 @@ static int rados_tool_common(const std::map < std::string, std::string > &opts,
 
     string oid(nargs[1]);
     string last_read = "";
-    int MAX_READ = 512;
+    int OMAP_BYTES_PER_REQUSET = MAX_OMAP_BYTES_PER_REQUSET;
```

Comment:
why not just use `MAX_OMAP_BYTES_PER_REQUSET`?

Reply:
well, you mean using same name for gobal variable and local variable？

Reply:
@dongbula the latter

Reply:
ok, have changed it, thanks